py/src/call.py

```diff
@@ -267,8 +324,14 @@ def resolve_block(connection, at_block):
         raise NoSuchBlock(at_block)
 
     gas_price = int.from_bytes(gas_price, "big")
+
+    if forced_gas_price is not None:
+        # allow caller to override any
+        gas_price = forced_gas_price
+
     sequencer_address = int.from_bytes(sequencer_address, "big")
 
+    # TODO: eth gas_price needs to be added, received from command
```

Comment:
It seems we already have `gas_price`? Or do I misunderstand this comment?

Reply:
Forgotten, the `forced_gas_price` came out of writing this. Will remove it.